### DISCUSSION TITLE
Player chrome

### DISCUSSION THREAD

**Discussion by heff** [luwes/playerx#16] [General]
Hi, thanks for the Mux shoutout!

I love the web component approach here. We've got a project that's probably very complimentary. https://github.com/muxinc/player-chrome

Cheers!
-heff

**Comment by luwes** [comment]
thanks for your message @heff, and for retweeting! :)

`player-chrome` looks very cool, custom skins is one of the goals for this project too. I like how every element is nicely isolated, makes it really easy to customize.

I hope I can work on the skins soon, right now I'm working on a few plugins that show a preview and handle lazy loading, something similar as https://github.com/paulirish/lite-youtube-embed and https://github.com/luwes/lite-vimeo-embed
